src/taker/offers.rs

```diff
@@ -32,6 +32,19 @@ pub struct OfferAndAddress {
     pub address: MakerAddress,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+enum Request {
+    Post { metadata: Box<OfferMetadata> },
+    Get { makers: u32 },
+    Dummy { url: String },
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+struct OfferMetadata {
+    url: String,
+    proof: FidelityProof,
+}
```

Comment:
These structures are used for all the systems. So lets put them somewhere common.

Reply:
Yes utills..